it/src/test/java/apoc/it/core/StartupTest.java

```diff
@@ -75,8 +75,8 @@ public void compare_with_sources() {
                             .list(record -> record.get("name").asString());
 
 
-                    assertEquals(sorted(ApocSignatures.PROCEDURES), procedureNames);
-                    assertEquals(sorted(ApocSignatures.FUNCTIONS), functionNames);
+                    assertEquals(sorted(ApocSignaturesCore.PROCEDURES), procedureNames);
```

Comment:
I had a brief look at your extended PR too. How come we test both core and extended in extended PR? I would (maybe naively) have thought that it would make more sense to have the checkCoreWithExtraDependenciesJars test here and the other 2 tests in the extended repo?

Reply:
Ah ok. I don't think I actually have tried to build them myself, so you might be right about that. If it was a conscious decision, I think it is ok.